frontend/src/views/DatabaseDetail.vue

```diff
@@ -143,7 +143,10 @@
             />
           </button>
           <button
-            v-if="allowAlterSchemaOrChangeData"
+            v-if="
+              allowAlterSchemaOrChangeData &&
+              database.instance.engine !== 'MONGODB'
```

Comment:
Maybe put it into `allowAlterSchemaOrChangeData`

Reply:
Agree, we may split `allowAlterSchemaOrChangeData` to `allowAlterSchema` and `allowChangeData` seperately